tests/db_functions/text/test_concat.py

```diff
@@ -21,13 +21,14 @@ def test_basic(self):
         Author.objects.create(name="Margaret", goes_by="Maggie")
         Author.objects.create(name="Rhonda", alias="adnohR")
         authors = Author.objects.annotate(joined=Concat("alias", "goes_by"))
+        #In Singlestore , If any of the arguments is NULL, the result is NULL in CONCAT.
```

Comment:
I think we should add `as_singlestore` to the `class ConcatPair(Func):`, see `as_mysql` there, it should work.